.travis.yml

```diff
@@ -65,3 +65,15 @@ script:
 
   # Code coverage
   - coverage report
+
+
+
+notifications:
+  slack: andela:l6HtahN5TFwe7CaKTqxjFCrJ
+
+# After the build test with coveralls
+after_success:
+- coveralls
+
+
```

Comment:
@andela-bmosigisi yes I fixed that, I set cycle CI not to follow the project too.but it is still building